src/core/router/routes-wallet.js

```diff
@@ -44,7 +44,7 @@ export default {
       beforeEnter: swapRouterGuard
     },
     {
-      path: 'dapps',
+      path: 'DApps',
```

Comment:
can you revert this back please? this is actual route name and they all should be lower case